parsl/tests/configs/user_opts.py

```diff
@@ -21,7 +21,24 @@
 # COMET_USERNAME = "yadunand"
 
 user_opts = {
-    # "public_ip" : PUBLIC_IP,
+    'frontera': {
+        'worker_init': 'source ~/setup_parsl_test_env.sh;',
+    },
+    'theta': {
+        'worker_init': 'source ~/setup_parsl_test_env.sh;',
+    },
+    'cori': {
+        'worker_init': 'source ~/setup_parsl_test_env.sh;',
+    },
+    'summit': {
+        'worker_init': 'source ~/setup_parsl_test_env.sh;',
+    },
+    'midway': {
+        'worker_init': 'source ~/setup_parsl_test_env.sh;',
+    },
+    'petrelkube': {
+        'worker_init': '~/setup_parsl_test_env.sh',
+    },
```

Comment:
@yadudoc I think you missed the user opts of `in2p3` here

Reply:
These need to be edited by the user anyway, and I have mine in a local_user_opts.